financial-templates-lib/logger/Transports.js

```diff
@@ -0,0 +1,53 @@
+// This module generates transport objects for the winston logger to push messages to.
+// Primarily this module separates the logic for reading in state environment variables
+// from the logger itself. All Winston transport objects and their associated formatting
+// are created within this module.
+
+// Transport objects
+const ConsoleTransport = require("./ConsoleTransport");
+const SlackTransport = require("./SlackTransport");
+const TwilioTransport = require("./TwilioTransport");
+
+require("dotenv").config();
+const argv = require("minimist")(process.argv.slice(), {});
+
+// transports array to store all winston transports
+let transports = [];
+
+// add a console transport to log to the console.
+transports.push(ConsoleTransport.createConsoleTransport());
+
+// If there is "test" in the environment then skip the slack or twilio transports.
+if (argv._.indexOf("test") == -1) {
```

Comment:
Is this the only changed part in `Transports.js`? I do like this touch, but is there a way we can "unit test" the Slack/Twilio transports?

Reply:
right now no. we'd need to write separate tests for these transport unfortunately. these would need to be full integration tests which are hard to do. for example they'd need to test the output of the web call to slack from the transport. Wed perhaps need to mock the slack transport to test what it sends out.